tests/Unit/SubPageList/Lister/SimpleSubPageFinderTest.php

```diff
@@ -47,5 +62,26 @@ public function testGetSubPagesFor( Title $title ) {
 		$this->assertInternalType( 'array', $pages );
 		$this->assertContainsOnlyInstancesOf( 'Title', $pages );
 	}
+	
+	/**
+	 * @dataProvider redirectProvider
+	 *
+	 * @param Title $title
+	 */
+	public function testGetSubPagesForRedirect( Title $title ) {
+		$finder = $this->newInstance();
+
+		$pages = $finder->getSubPagesFor( $title );
+
+		$this->assertInternalType( 'array', $pages );
+		$this->assertContainsOnlyInstancesOf( 'Title', $pages );
```

Comment:
As of PHP 5.5, which is now the min PHP version this extension supports, you can reference classes using `::class`, and not need strings. See at the bottom here https://www.entropywins.wtf/blog/2016/01/30/missing-in-php7-function-references/